config/application.rb

```diff
@@ -16,6 +16,8 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.to_prepare do
+      Qa::Authorities::Local::FileBasedAuthority.prepend ::PrependFileBasedAuthority
```

Comment:
this line was in "config/initializers/alta.rb", which we have removed to DRY up the code.